### ISSUE TITLE
Transactional Emails have nil pointer

### ISSUE THREAD

**Issue by danquack** [joeirimpan/listmonk-messenger#42]
```
POST /api/tx
{
      "subscriber_email": "me@mydomain.com",
      "template_id": 8,
      "content_type": "html",
      "messenger": "ses",
      "from_email": "sender@mydomain.com"
  }
```

```
2023/12/03 21:55:22 http: panic serving 10.0.2.3:52728: runtime error: invalid memory address or nil pointer dereference
goroutine 44 [running]:
net/http.(*conn).serve.func1()
	/usr/local/go/src/net/http/server.go:1825 +0xbf
panic({0x9864e0, 0x101e370})
	/usr/local/go/src/runtime/panic.go:844 +0x258
github.com/joeirimpan/listmonk-messenger/messenger.sesMessenger.Push({{{_, _}, {_, _}, {_, _}, _}, _, _}, {{0x0, ...}, ...})
	/app/messenger/ses.go:57 +0x1a2
main.handlePostback({0xd1b798, 0xc0005b80e0}, 0xc0001a2900)
	/app/handlers.go:128 +0xd28
net/http.HandlerFunc.ServeHTTP(...)
	/usr/local/go/src/net/http/server.go:2084
main.wrap.func1({0xd1b798, 0xc0005b80e0}, 0xc0001a2800)
	/app/handlers.go:142 +0x253
net/http.HandlerFunc.ServeHTTP(0x982f60?, {0xd1b798?, 0xc0005b80e0?}, 0xc0005a6105?)
	/usr/local/go/src/net/http/server.go:2084 +0x2f
github.com/go-chi/chi.(*Mux).routeHTTP(0xc000076ae0, {0xd1b798, 0xc0005b80e0}, 0xc0001a2800)
	/go/pkg/mod/github.com/go-chi/chi@v4.1.2+incompatible/mux.go:431 +0x1f9
net/http.HandlerFunc.ServeHTTP(0xd1ba78?, {0xd1b798?, 0xc0005b80e0?}, 0x101e190?)
	/usr/local/go/src/net/http/server.go:2084 +0x2f
github.com/go-chi/chi.(*Mux).ServeHTTP(0xc000076ae0, {0xd1b798, 0xc0005b80e0}, 0xc0001a2700)
	/go/pkg/mod/github.com/go-chi/chi@v4.1.2+incompatible/mux.go:86 +0x3c2
net/http.serverHandler.ServeHTTP({0xd1a490?}, {0xd1b798, 0xc0005b80e0}, 0xc0001a2700)
	/usr/local/go/src/net/http/server.go:2916 +0x43b
net/http.(*conn).serve(0xc00075a0a0, {0xd1bb20, 0xc0005a4750})
	/usr/local/go/src/net/http/server.go:1966 +0x5d7
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:3071 +0x4db
```

**Comment by joeirimpan** [comment]
Hi @danquack ,

While reviewing the code in Listmonk, I noticed that transactional postbacks do not include a "from_email" address. This can cause a nil pointer error when dereferencing the campaign. The issue lies in the [postback struct](https://github.com/knadh/listmonk/blob/master/internal/messenger/postback/postback.go#L97), which does not have a "from_email" field apart from the one set within the campaign. To address this, a potential solution would be to add a "from_email" field to the postback struct and populate it from the relevant location indicated [here](https://github.com/knadh/listmonk/blob/master/cmd/tx.go#L127). By making this modification, the issue should be resolved.

**Comment by joeirimpan** [comment]
This is fixed in [v0.0.4](https://github.com/joeirimpan/listmonk-messenger/releases/tag/v0.0.4) and the `master` branch of [listmonk](https://github.com/knadh/listmonk/commits/master)
